src/components/common/Map.js

```diff
@@ -1,31 +1,37 @@
 import React from 'react';
 import { withScriptjs, withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
-import { bool } from 'prop-types';
+import { bool, number } from 'prop-types';
 
 import marker from '../../img/marker.svg';
 
-const Map = withScriptjs(withGoogleMap((props) => {
-  let latitude;
-  let longitude;
-  navigator.geolocation.getCurrentPosition((position) => {
-    ({ latitude, longitude } = position.coords);
-  });
-  const { isMarkerShown } = props;
+const Map = withScriptjs(withGoogleMap(({ handleClick, isMarkerShown,
```

Comment:
Is `withScriptjs` necessary?

Also, can we move those decorators to the bottom of the file, where you export the component, to keep this cleaner?

It would be something like `export default withScriptjs(withGoogleMap((Map)));`